app/controllers/items_controller.rb

```diff
@@ -1,20 +1,21 @@
 class ItemsController < ApplicationController
-  def home
-    @item = Item.new
-    @items = Item.all
-  end
 
   def create
-    @item = Item.new(item_params)
+    @inventory_dashboard = InventoryDashboard.new(Item.all)
 
-    if @item.save
+    if @inventory_dashboard.save_item(item_params)
+      flash[:success] = 'success'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.